changelog/13650.bugfix.rst

```diff
@@ -0,0 +1 @@
+Restore pre-ExceptionGroup semantics for CallInfo.from_call: re-raise exceptions in reraise even when they appear inside a single-level ExceptionGroup. Nested groups remain out of scope of this fix and behave as before.
```

Comment:
Thanks for the changelog, but this contains too much internal details that are not relevant for users.

I suggest:


```suggestion
Fixed an issue where internal exceptions like those raised from :func:`pytest.exit` would not be correctly handled during test teardown.
```